### PR TITLE
Write export files as xml

### PR DESCRIPTION
**Pull request by norkans7**
- [x] Check the download link

- [x] Check the extension and format of the download field, make sure the downloaded file is decompressed as xml

### CODE REVIEW

File: temba/utils/export.py [hunk 1]
```diff
@@ -180,31 +180,50 @@ def __init__(self, task, sheet_name, extra_sheet_name, columns, extra_columns):
...
-        self.sheet_row = 2
+        self.sheet_row += 1
 
     def write_row(self, values, extra_values):
         """
         Writes the passed in row to our exporter, taking care of creating new sheets if necessary
         """
         # time for a new sheet? do it
-        if self.sheet_row > BaseExportTask.MAX_EXCEL_ROWS:
+        if self.sheet_row >= BaseExportTask.MAX_EXCEL_ROWS:
```
Comment (nicpottier) [inline temba/utils/export.py]: What is max rows?

Comment (norkans7) [reply temba/utils/export.py]: 1,048,576

**Comment by norkans7** [comment]
I thought it is easier to download a smaller file and unzip . the xml is really big and might be harder to download on some connections

**Comment by nicpottier** [comment]
If you set the headers correctly on the S3 file, it will be unzipped by the browser. IE, upload a gzip with content-encoding set to gzip. It will appear on the user's machine as XML but will be transferred as a gzip.

**Comment by nicpottier** [comment]
For kicks I played around with the full xls format and I don't think it would be too bad to recreate these entirely, it is just a bit more boilerplate.

You can rename `xls` files to `zip` and then extract them, for an empty Excel file with one sheet you get:

```
Archive:  test_xls.zip
  inflating: [Content_Types].xml
  inflating: _rels/.rels
  inflating: xl/_rels/workbook.xml.rels
  inflating: xl/workbook.xml
  inflating: xl/styles.xml
  inflating: xl/theme/theme1.xml
  inflating: xl/worksheets/sheet1.xml
  inflating: docProps/core.xml
  inflating: docProps/app.xml
```
Of those I'm pretty sure styles and theme can be removed, and the rest is easy boilerplate. It follows a similar pattern as this XML format as the sheets are in separate files, we just need to create the other files around it and zip the whole thing, renaming it to XLS. 

Here's what one of the sheets looks like for example:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="x14ac xr xr2 xr3" xmlns:x14ac="http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac" xmlns:xr="http://schemas.microsoft.com/office/spreadsheetml/2014/revision" xmlns:xr2="http://schemas.microsoft.com/office/spreadsheetml/2015/revision2" xmlns:xr3="http://schemas.microsoft.com/office/spreadsheetml/2016/revision3" xr:uid="{852FBDB0-629E-1946-BE40-5DC23BEC3B9F}"><dimension ref="A1:D1"/><sheetViews><sheetView tabSelected="1" workbookViewId="0"><selection activeCell="D1" sqref="D1"/></sheetView></sheetViews><sheetFormatPr baseColWidth="10" defaultRowHeight="16" x14ac:dyDescent="0.2"/><sheetData><row r="1" spans="1:4" x14ac:dyDescent="0.2"><c r="A1"><v>4</v></c><c r="B1"><v>5</v></c><c r="C1"><v>6</v></c><c r="D1"><v>6</v></c></row></sheetData><pageMargins left="0.7" right="0.7" top="0.75" bottom="0.75" header="0.3" footer="0.3"/></worksheet>
```

**Comment by norkans7** [comment]
The xml generated for many cons\tacts is too big that excel does not open it.
This needs to use xlsx format

**Comment by nicpottier** [comment]
Huh, interesting.. so what about trying to fill out the boilerplate to write a real xlxs file? Seems none of the libraries are fast enough right?

**Comment by norkans7** [comment]
yep, still checking how they reference each other

**Comment by norkans7** [comment]
Replaced by https://github.com/nyaruka/rapidpro/pull/1962

**Pull request closed without merging**